content/zh/blog/exceptions.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Falco Rules Now Support Exceptions
```

Comment:
You can translate `Falco Rules Now Support Exceptions` if you want, just keep the key `title` as it